pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -147,6 +153,10 @@ class TestGroupby(BaseDecimal, base.BaseGroupbyTests):
     pass
 
 
+# TODO(extension)
+@pytest.mark.xfail(reason=(
+    "raising AssertionError as this is not implemented, "
+    "though easy enough to do"))
```

Comment:
I think you can change the AssertionError to a ValueError in the code, and then we can still test this.